websites/S/Spotify Podcasts/dist/metadata.json

```diff
@@ -4,6 +4,9 @@
     "id": "241278257335500811"
   },
   "service": "Spotify Podcasts",
+  "altnames": [
+    "스포티파이 팟캐스트"
```

Comment:
you speak korean ye? since when

Reply:
Been learning it for a while, didn't say I was good or dedicated to learning it lmao